_posts/en/2023-10-22-kafka-1.6.md

```diff
@@ -0,0 +1,61 @@
+---
+layout:    post
+title:     "Kouncil - permissions"
+date:      2023-10-22 6:00:00 +0100
+published: true
+didyouknow: false
+lang: en
+lang-ref:  kouncil-1.6
+interpreter: Justyna Wiatrzyk-Guzik
+author:    pbelke
+image:     /assets/img/posts/2023-10-22-kouncil-1.6/kouncil-1.6.png
+description: "The latest version of Kouncil introduces permission configuration, allowing for precise control over user access to various system functions."
+tags:
+- kouncil
+- programming
+- kafka
+- authentication
+- brokers
+---
+
+The most important change in the new version of Kouncil is the configuration of permissions, which allows precise control of user access to particular functions. In addition, in an effort to continuously improve the application's interface, we have introduced several minor changes to make the system easier to work with.
+
+## Permissions configuration
+Thanks to a new function for granting permissions, a logged-in user has access to all Kouncil functions defined for their assigned role.
+
+In the latest Kouncil version, users are assigned to respective roles that determine their access to functions:
+* users with administrative roles gain access to broker and consumer group screens
+* users with the editor role can view topic screens, track messages, and send communications to selected topics
+* users with read-only permission can view messages and data on the topic screens
+
+The flexible permission structure enables Kouncil's functionality to be effectively adapted to the needs and responsibilities of users in different positions.
+
+The screens below are sample views for particular user groups.
+
+![2019-09-05-Kafka-DLQ-Strategy01.png](/assets/img/posts/2023-10-22-kouncil-1.6/kouncil-1.6-1.png)
+*Screen for users with administrator’s permissions in the default configuration*
+
+![2019-09-05-Kafka-DLQ-Strategy01.png](/assets/img/posts/2023-10-22-kouncil-1.6/kouncil-1.6-2.png)
+*Screen for users with editor’s permissions in the default configuration*
+
+![2019-09-05-Kafka-DLQ-Strategy01.png](/assets/img/posts/2023-10-22-kouncil-1.6/kouncil-1.6-3.png)
+*Screen for users with read-only permissions in the default configuration*
+
+If a user tries to access a page for which they do not have permissions, a screen with access denial will be displayed.
+
+![2019-09-05-Kafka-DLQ-Strategy01.png](/assets/img/posts/2023-10-22-kouncil-1.6/kouncil-1.6-4.png)
+*Screen notifying users of the lack of permission and refusal of access to the function*
+
+Configuring permissions involves assigning roles from the selected user authentication method to roles in Kouncil.
+
+## Effect
```

Comment:
Nie używamy nagłówków do formatowania tekstu. Nagłówki są używane do nawigacji po tekście. Dodatkowo, dzięki nim zewnętrzne witryny (np wyszukiwarki) mogą odsyłać do konkretnego miejsca.

Prawdopodobnie to co chciałeś osiągnąć możesz zrobić za pomocą takiego kodu:
```
## Effect
**As a result, you can grant users access to role-specific functions.**
```